.github/workflows/general.yml

```diff
@@ -157,7 +157,7 @@ jobs:
         uses: flutter-actions/setup-flutter@v2
         with:
           channel: stable
-          version: 3.24.2
+          version: 3.29.3
```

Comment:
we should use the same version in both places?  not 3.29 and 3.27?

Reply:
Yes.  I should fix.

I tested 3.29 and it fails the analysis stage because dart:html is now fully deprecated and the changes look really painful (I tried to convert but got lost).  3.27 fixes the weird shadow bugs in the display, but without requiring dart:html to be removed (the new methodology is package:web).  It gets really weird because they changed WebASM and the Worker class, so there are tons of problems, not just renamings (e.g. a lot of things just moved to web.*  but the WebASM stuff is quite different).